### ISSUE TITLE
Any way to create conditional fields

### ISSUE THREAD

**Issue by elpeyotl** [agentejo/cockpit#669]
Hello guys!

First thank your for this fine little piece of software. Started to digging into it, and I'm always finding out new stuff. Very nice. 

Is there a way to create conditional fields? I mean when an editor is entering content and he activates a boolean or checkbox, then the page will show him some other fields? Anyway this would be a very nice feature and complete this cms. 

Best

**Comment by aheinze** [comment]
no, not yet. but I already thought about implementing this feature ☝️ we'll see ;-)

**Comment by meadleyson** [comment]
+1

**Comment by KeitIG** [comment]
Same here, would luv to see that, maybe have a look at [ACF](https://www.advancedcustomfields.com/) for inspiration :)

**Comment by Blazsoul** [comment]
This feature could really help me out
I tried to implement this behavior with repeater, the problem is that i cannot limit the number of items in repeater to 1

**Comment by lucasvignolireis** [comment]
+1 would love this also!
